cmdx.py

```diff
@@ -357,11 +358,16 @@ def __call__(cls, mobject, exists=True, modifier=None):
         # It didn't exist, let's create one
         # But first, make sure we instantiate the right type
         if mobject.hasFn(om.MFn.kDagNode):
-            sup = DagNode
+            if mobject.hasFn(om.MFn.kMesh):
+                sup = Mesh
+            else:
+                sup = DagNode
```

Comment:
Could we flatten this if branch?

```py
        if mobject.hasFn(om.MFn.kMesh):
            sup = Mesh
        elif mobject.hasFn(om.MFn.kDagNode):
            sup = DagNode
        elif mobject.hasFn(om.MFn.kSet):
            sup = ObjectSet
        ...
```

Reply:
I agree for the readabilty. I did this to gain some µs from the MFn check of shape which is useless once you did the one for DagNode. The point was to reproduce the inheritance of classes there